libobs/obs-output.h

```diff
@@ -44,6 +45,9 @@ struct obs_output_info {
 
 	uint32_t flags;
 
+	/* Broadcast Performance Metrics control */
+	bool enable_bpm;
+
```

Comment:
Info structure should be considered non-mutable once registered.

```suggestion
```

Reply:
Thanks for suggestion tytan652. If the `bool enable_bpm` field is removed, should I use the `flags` field in `struct obs_output_info` instead to store the BPM enable bit?

Reply:
@tytan652 I moved the `enable_bpm` boolean to the `obs_output` struct instead of the `obs_output_info` struct and adjusted references accordingly. Please let me know if this make sense.

Reply:
> Please let me know if this make sense.

Implementing this inside of libobs does not make sense at all for me since it's using SEI metadata to send custom information, but I needed second opinions which @tt2468 gave.